tests/integrationv2/test_renegotiate_apache.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import tempfile
+
+from configuration import ALL_TEST_CURVES
+from common import ProviderOptions
+from fixtures import managed_process
+from providers import Provider, S2N
+from utils import invalid_test_parameters, get_parameter_name
+from constants import TEST_CERT_DIRECTORY
+
+from test_renegotiate import TEST_PROTCOLS, S2N_RENEG_OPTION, S2N_RENEG_ACCEPT
```

Comment:
I hate to make you fix my typos, but...
```suggestion

from test_renegotiate import TEST_PROTOCOLS, S2N_RENEG_OPTION, S2N_RENEG_ACCEPT
```

Reply:
Fixed! 271bdf0914434cd8a9c7949f572660c884abed18